packages/manager/apps/hpc-vmware-vsphere/src/pages/dashboard/index.tsx

```diff
@@ -47,15 +93,53 @@ export default function DashboardPage() {
       .reverse()
       .find((tab) => location.pathname.startsWith(tab.to));
     if (activeTab) {
-      setActivePanel(activeTab.to);
+      if (activeTab.isRedirectLegacy) {
+        const loadLegacyApp = async () => {
+          const legacyAppUrl = (await shellNav.getURL(
+            'dedicated',
+            `#/dedicated_cloud/${serviceName ?? ''}/${activeTab.to}`,
+            {},
+          )) as string;
+          window.location.href = legacyAppUrl;
+        };
+        loadLegacyApp();
+      } else {
+        setActivePanel(activeTab.to);
+      }
     } else {
       setActivePanel(tabsList[0].to);
       navigate(`${tabsList[0].to}`);
     }
   }, [location.pathname]);
 
-  const header = {
-    title: t('title'),
+  const changelogsLink: ChangelogLinks = {
+    changelog:
+      'https://github.com/orgs/ovh/projects/16/views/6?sliceBy%5Bvalue%5D=VMware+on+OVHcloud',
+    roadmap:
+      'https://github.com/orgs/ovh/projects/16/views/1?pane=info&sliceBy%5Bvalue%5D=VMware+on+OVHcloud',
+    'feature-request':
+      'https://github.com/ovh/infrastructure-roadmap/issues/new?assignees=&labels=&projects=&template=feature_request.md&title=',
+  };
```

Comment:
May be can be declared externally of component. It's a details but this allow to not create new reference between 2 re-render and it's free